pages/advanced/cli-reference/tx-service-operations.mdx

```diff
@@ -0,0 +1,100 @@
+import { Callout } from 'nextra/components'
+
+# Operations for the `tx-service` mode
+
+## Return balance
```

Comment:
Inconsistency here:
- Return (the) balance (of a Safe)
- Execute **a** pending transaction
- Sign (a) transaction
- Etc.